compiler/wasm/test/shared.ts

```diff
@@ -1,4 +1,4 @@
-import { initialiseResolver } from "@noir-lang/noir-source-resolver";
+import { initialiseResolver } from "@noir-lang/source-resolver";
```

Comment:
```suggestion
import { initializeResolver } from "@noir-lang/source-resolver";
```

Doesn't have to be done in this PR but if we're switching the packages then let's yeehawify it.

Reply:
just yeehawed the codebase